client/app/templates/candidates/candidate.hbs

```diff
@@ -0,0 +1,10 @@
+{{page-title (t "page_titles.candidate")}}
+
+<h1>{{t "page_titles.candidate"}}</h1>
+
+<ApplicantForm
+    @applicantAge={{this.model.age}}
+    @applicantName={{this.model.name}}
```

Comment:
Passing these individually as `args` are watched by default. Just passing the `applicant` would require jumping through a few more hoops to activate observers.